qiskit_machine_learning/algorithms/classifiers/pegasos_qsvc.py

```diff
@@ -67,8 +67,9 @@ def __init__(
     ) -> None:
         """
         Args:
-            quantum_kernel: a quantum kernel to be used for classification. Has to be ``None`` when
-                a precomputed kernel is used.
+            quantum_kernel: A subclass of :class:`~qiskit_machine_learning.kernels.BaseKernel`
```

Comment:
Having the BaseKernel linked on this parameter, when the parameter types are included in the docs from the typehint on the parameter seems like its unnecessary. This is how the docs look now and to me the typehint there already says it needs to be of type BaseKernel and is a link (you can see this in the existing docs too).

I will only comment on this one but there are more instances. My take would be to not have duplication like this.

![image](https://github.com/qiskit-community/qiskit-machine-learning/assets/40241007/e35ca4b1-c6e0-49c6-90ba-f41792646a1e)

Having it duplicated when it's brought in from the typehint and linked via that seems like

Reply:
Reset the first sentence to the original and add the additional note that the quantum kernel defaults to FidelityQuantumKernel if precomputed is `False` and quantum_kernel is `None`.